src/libs/DistanceRequestUtils.ts

```diff
@@ -122,14 +185,50 @@ function getDistanceMerchant(
         return translate('iou.routePending');
     }
 
-    const formattedDistance = getDistanceForDisplay(hasRoute, distanceInMeters, unit, rate, translate);
-    const singularDistanceUnit = unit === CONST.CUSTOM_UNITS.DISTANCE_UNIT_MILES ? translate('common.mile') : translate('common.kilometer');
-    const ratePerUnit = PolicyUtils.getUnitRateValue(toLocaleDigit, {rate});
+    const distanceInUnits = getDistanceForDisplay(hasRoute, distanceInMeters, unit, rate, translate);
+    const ratePerUnit = getRateForDisplay(unit, rate, currency, translate, toLocaleDigit);
```

Comment:
NAB: Can we call this formattedRate instead, or something similar? Above we call a different function for ratePerUnit